### PR TITLE
[IMPROVEMENT] Add introductory image to SRP Quiz

### PR DESCRIPTION
**Pull request by gantunesr**
**Description**

_Write a short description of the changes included in this pull request, also include relevant motivation and context. Have in mind the following questions,_

Add introductory image to SRP Quiz.

**Screenshots/Recordings**

1. Android - Dark Mode

2. iOS - Light Mode

**Issue**

Progresses https://github.com/MetaMask/mobile-planning/issues/591

**Checklist**

* [x] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

### CODE REVIEW

File: app/components/Views/Quiz/SRPQuiz/SRPQuiz.tsx [hunk 1]
```diff
@@ -18,6 +19,8 @@ import { QuizStage } from '../types';
 import { QuizContent } from '../QuizContent';
 import stylesheet from './styles';
 
+const introductionImg = require('../../../../images/reveal-srp.png');
```
Comment (blackdevelopa) [inline app/components/Views/Quiz/SRPQuiz/SRPQuiz.tsx:22]: Can we get the svg for this?

Comment (gantunesr) [reply app/components/Views/Quiz/SRPQuiz/SRPQuiz.tsx:22]: We don't use SVGs in the app, all the images are PNGs. A change we can discuss in the next mobile community sync c @sethkfman

Comment (blackdevelopa) [reply app/components/Views/Quiz/SRPQuiz/SRPQuiz.tsx:22]: Oh yeah, you are right. I think at this time, it's only used in the component library, [here](https://github.com/MetaMask/metamask-mobile/blob/8690260f43383a9458a27b169e5211ce6d940359/app/component-library/components/Icon/Icon.tsx#L30).

Comment (blackdevelopa) [reply app/components/Views/Quiz/SRPQuiz/SRPQuiz.tsx:22]: Do you think we can reduce the size of the png file?

Comment (gantunesr) [reply app/components/Views/Quiz/SRPQuiz/SRPQuiz.tsx:22]: I think so, which size you have in mind?

Comment (gantunesr) [reply app/components/Views/Quiz/SRPQuiz/SRPQuiz.tsx:22]: [a20e387](https://github.com/MetaMask/metamask-mobile/pull/5647/commits/a20e38768383cc276327c12b9b26738031c2bf8c)

**Review by blackdevelopa — APPROVED** [review]
LGTM

**Comment by holantonela** [comment]
Looks awesome, thanks @gantunesr

**Pull request merged**